### DISCUSSION TITLE
Custom Ruleset Multiplayer Support

### DISCUSSION THREAD

**Discussion by TheFantasticLoki** [ppy/osu#16937] [Ideas] [answered]
Community Rulesets have come a LONG ways and I feel that we are at a point where multiplayer should be possible whilest using these custom rulesets especially seeing as we only have local leaderboards atm anyways.

**Comment by peppy** [comment, marked as answer]
There are prerequisites to make this happen, including some form of official listing and distribution. I agree that rulesets are ready, but we are not. There are higher priorities currently but I would hope that we can revisit this later this year.

Note that when we get to adding support, it wouldn't be for just multiplayer but also things like leaderboards.

**Reply by TheFantasticLoki** [reply]
In terms of official Listing and Distribution this comes pretty close: https://github.com/Rurusetto/rurusetto/blob/new-website/README.md

Included also in another Repo is the Addon for the Lazer Client which allows you to easily download and update your rulesets: https://github.com/Flutterish/rurusetto-addon/blob/main/README.md

**Reply by TheFantasticLoki** [reply]
Edit: switched screenshot to one taken on [2022.205.0](https://github.com/ppy/osu/releases/tag/2022.205.0) Which works for Rurusetto Addon V1.8
![osu_2022-02-20_21-14-05](https://user-images.githubusercontent.com/26889568/154888021-01ed8bf1-9295-4494-afcb-5345356bdbcc.png)

Included here is a screenshot of what it looks like in game tho due to the latest game update it's currently crashing and in need of an update itself.

**Reply by peppy** [reply]
External sites providing a listing do not help in any way or form.
